server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/src/test/java/org/apache/james/webadmin/integration/rabbitmq/RabbitMQWebAdminServerTaskSerializationIntegrationTest.java

```diff
@@ -511,15 +512,17 @@ void postRedeliverSingleEventShouldCreateATask(GuiceJamesServer guiceJamesServer
             .mailboxPath(MailboxPath.forUser(Username.of(USERNAME), "Important-mailbox"))
             .build();
 
-        guiceJamesServer
+        EventDeadLetters.InsertionId insertionId = guiceJamesServer
             .getProbe(EventDeadLettersProbe.class)
             .getEventDeadLetters()
             .store(group, event)
             .block();
 
+        Assertions.assertThat(insertionId).isNotNull();
```

Comment:
Please remove this assertion: it's not the main assertion, it makes the test case harder to read.